proposals/td_for_tdir.md

```diff
@@ -18,15 +18,22 @@ Over time the team [realised](https://github.com/mozilla-iot/schemas/issues/35)
 ### Example
 
 Example Thing Description for a gateway:
-```
+```json
 {
   "title": "WebThings Gateway",
   "@context": "https://iot.mozilla.org/schemas/",
   "@type": "Gateway",
   "actions": {
+    "startPairing": {
+      "name": "Start Pairing",
+      "@type": "StartPairingAction"
+    },
```

Comment:
This is a good solution to the add/pair problem, thank you.

I'm tempted to leave the StartPairingAction out of the example and just mention it in the comments below, in order to keep the Thing Description generic to any kind of directory, not just gateways. Then possibly change `@type` to `Directory` rather than `Gateway` to reflect that.

Alternatively it could be `"@type": ["Directory", "Gateway"]` and define two capability schemas (with `Gateway` being a specialisation of `Directory`), but that might just complicate matters.

Reply:
> I'm tempted to leave the `StartPairingAction` out of the example and just mention it in the comments below, in order to keep the Thing Description generic to any kind of directory, not just gateways.

I would agree with that.

> Then possibly change `@type` to `Directory` rather than `Gateway` to reflect that.

I agree that these are different types of devices, with gateway being a superset of directory.

> Alternatively it could be `"@type": ["Directory", "Gateway"]` and define two capability schemas (with `Gateway` being a specialisation of `Directory`), but that might just complicate matters.

A Gateway may just have some distinct actions, but not much else. It may be unnecessary to define it as a schema, though, as different gateways may have entirely different purposes.

Reply:
OK, cool, then lets change the `@type` to `Directory`, remove `StartPairingAction` from the example and just mention it in the note below.